### ISSUE TITLE
Deprecated package scrpage2 should be replaced by scrlayer-scrpage

### ISSUE THREAD

**Issue by frehage** [frehage/ThesisTemplate#2]
scrpage2 is used to create the nice page headers. We need to refactor this such that we can use scrlayer-scrpage instead.

**Comment by krooken** [comment]
This is fixed once pull request #9 is merged into this repo.

**Comment by frehage** [comment]
Pull request is now done. Thank you!
